src/patternfly/components/DataList/data-list.scss

```diff
@@ -467,7 +410,12 @@
 // expandable content
 .#{$data-list}__expandable-content {
   max-height: var(--#{$data-list}__expandable-content--MaxHeight);
+  margin-block-end: 16px;
+  margin-inline-start: 16px;
+  margin-inline-end: 16px;
```

Comment:
Can't recall if this was gone over during the Penta call, but should these properties be assigned variables instead of hardcoded values?